### DISCUSSION TITLE
Switching between different windows

### DISCUSSION THREAD

**Discussion by mandjeo** [rxhanson/Rectangle#1379] [General]
Hi, 
I'm a satisfied user of Rectangle Pro.
I have an idea/question is there a feature to have a shortcut to switch between any window of any app that is open.
So if I have multiple windows of the same app that I switch between those too (which is not provided by the normal Cmd + tab shortcut)?

It would be something similar to what Windows provides out of the box with Alt + tab.

In case that there's another app that provides this please let me know.

Regards,
Milos

**Comment by rxhanson** [comment]
Thanks for going with Rectangle Pro! 

You can switch between windows of the frontmost app by pressing cmd+`

Beyond that, perhaps this is what you're looking for: https://github.com/lwouis/alt-tab-macos

I do have plans to add my own implementation of window switching to Rectangle Pro in the future.

**Reply by mandjeo** [reply]
Thanks a lot for the feedback!
This app is what I was looking for, but of course, will be glad to switch to your alternative once you have it out.

Cheers,
Milos
